packages/clerk-js/src/core/clerk.ts

```diff
@@ -305,6 +330,14 @@ export default class Clerk implements ClerkInterface {
     this.#resetComponentsState();
   };
 
+  /** @deprecated Use `setActive` instead */
```

Comment:
Folks, `setSession` is a very critical function for ClerkJS. I'd suggest that we don't mark it as deprecated yet to avoid confusion. 

Moreover, as Alex said before, `setActive` requires necessary refactorings. So, we'd rather do all these at once.

We will deprecate it when we go to ClerkJS V4 in June.

Reply:
Alright, makes sense. I'll change it.